.ort.yml

```diff
@@ -148,7 +148,7 @@ curations:
     reason: "DOCUMENTATION_OF"
     comment: >-
       Documentation contains examples mentioning various licenses.
-  - path: "plugins/reporters/web-app-template/public/index.html"
+  - path: "plugins/reporters/web-app-template/index.html"
```

Comment:
> Why are all .js now .jsx files? And what are .jsx files after all?

*.js is plain javascript file whilst *.jsx are JSX files. JSX is a syntax extension for JavaScript coming from React project that lets HTML-like markup inside a JavaScript file. Not all *.js are *.jsx files now only those of the UI which have HTML within JavaScript are now JSX file. For Vite to work it requires correct extensions.

Reply:
> Why was logo.svg deleted?

Logo.svg actually contained the React logo (see https://github.com/oss-review-toolkit/ort/blob/c7d5c3a4/plugins/reporters/web-app-template/src/logo.svg), it was never used within the WebApp but was there to prevent create-react-app from throwing an error during build that logo.svg was missing. As we now migrate off create-react-app to vite logo.svg can savely be deleted.